trussworks/react-uswds
# docs: adds in information on how to test PRs in GitHub Codespaces

# Summary

Adds info doc about using Codespaces to review PRs so you can check out changes and run them on a dev server without having to on your machine, reducing context switching and lowering the barrier to PR reviews.

Adds reference in design PR review doc to this page

## Related Issues or PRs

[ docs: add design PR review documentation #2615 ](https://github.com/trussworks/react-uswds/pull/2615)

## How To Test

Int he changed files view, click the icon in the upper right of the md file for `Display rich diff`. This will let you preview the markdown file. Follow the steps in the process and make sure I have outlined the process well enough for you to figure it out.

<img width="1324" alt="image" src="https://github.com/trussworks/react-uswds/assets/59394696/6bfffa9d-38ad-4f73-abb8-1c769a6912f4">
